pkg/credentialquery/credentialquery.go

```diff
@@ -26,8 +41,18 @@ func NewInstance(documentLoader ld.DocumentLoader) *Instance {
 // Query returns credentials that match PresentationDefinition.
 func (c *Instance) Query(
 	query *presexch.PresentationDefinition,
-	contents []*verifiable.Credential,
+	contents Credentials,
```

Comment:
The `Credentials` argument needs to be a pointer, otherwise `gomobile` will skip generating bindings for this method